components/gcp/core/kfp/gcp/core/base_op.py

```diff
@@ -0,0 +1,148 @@
+import signal
+import os
+from kubernetes import client, config
+from kubernetes.client.rest import ApiException
+from google.cloud import storage
+import google
+import json
+from datetime import datetime
+import logging
+import sys
+
+class BaseOp:
```

Comment:
What I do with comments are:

Any public classes or functions (can be accessed without "_") need to have full docstring (short desc, long desc, args, returns, raises). Any internal classes or functions without a leading "_" should have at least a short desc.

Reply:
Will do